### ISSUE TITLE
Bug: cant get `defineExpose` data which nested in component

### ISSUE THREAD

**Issue by priority3** [vuejs/test-utils#1855]
I have the component named `Collapse` and child component named `CollapseItem`

and the `CollapseItem` has the `defineExpose`
```js
// CollapseItem.vue
<script setup lang="ts" >
const isActive = computed(() =>
  collapse?.activeNames.value.includes(props.name),
)
defineExpose({
  isActive,
})
```

**Describe the bug**

```js
// I register `collapse-item` in the global
import {config,mount} from '@vue/test-utils'
import {Collapse,CollapseItem} from 'xxx'
config.global.stubs = {
	CollapseItem,
} 
const wrapper = mount(Collapse,{
	props: {
		activeNames: ['1']	
	},
	slots: {
		default:`
          <collapse-item name="1" title="title1">111</collapse-item>
          <collapse-item name="2" disabled>222</collapse-item>
        `,
	}
})
const vm = wrapper.vm
const collapseWrapper = wrapper.findComponent(OCollapse)
const collapseItemWrappers = collapseWrapper.findAllComponents(
  OCollapseItem,
) as VueWrapper<CollapseItemInstance>[]


// BUG there is fail, `isActive` is not in vm 
expect(collapseItemWrappers[0].vm.isActive).toBe(true)
```
![image](https://user-images.githubusercontent.com/75966543/201296913-9a4b5376-8a49-43b5-94c3-1b63017b2d22.png)

the child component `CollapseItem` is just get props
![image](https://user-images.githubusercontent.com/75966543/201297128-a49cba3d-8def-4f9c-8a88-72ae9af98e71.png)

**To Reproduce**

**Expected behavior**

**Related information:**
- `@vue/test-utils` version: ^2.1.0 
- `Vue` version: ^3.2.41
- `node` version: v16.14.2
- `npm` (or `yarn`) version: 8.15.0
- `pnpm` version: 7.13.4

**Additional context**

**Comment by cexbrayat** [comment]
Hi @priority3 

Can you make a small online repro so we can take a look? https://stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest__

**Comment by priority3** [comment]
> Hi @priority3
> 
> Can you make a small online repro so we can take a look? [stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest](https://stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest)__

oh, thanks for your reply,I had found that maybe this is my bad in coding by test

**Comment by spacedawwwg** [comment]
> > Hi @priority3
> > Can you make a small online repro so we can take a look? [stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest](https://stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest)__
> 
> oh, thanks for your reply,I had found that maybe this is my bad in coding by test

@priority3 What did you do to fix/change/workaround this, as I have hit the same issue that doing:

```
defineExpose({ videoRef })
```

and then using:

```
wrapper.vm.videoRef
```

No longer works

**Comment by cexbrayat** [comment]
I think I know what's going on: there is a breaking change in Vue core v3.2.45, and we need to release a new version of VTU to work around it (see #1858 ). 

You can stay on v3.2.44 if you are encountering this issue with VTU v2.2.1

**Comment by priority3** [comment]
> > > Hi @priority3
> > > Can you make a small online repro so we can take a look? [stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest](https://stackblitz.com/edit/vitest-dev-vitest-ledp6h?file=package.json&initialPath=__vitest)__
> > 
> > 
> > oh, thanks for your reply,I had found that maybe this is my bad in coding by test
> 
> @priority3 What did you do to fix/change/workaround this, as I have hit the same issue that doing:
> 
> ```
> defineExpose({ videoRef })
> ```
> 
> and then using:
> 
> ```
> wrapper.vm.videoRef
> ```
> 
> No longer works

@spacedawwwg , it's strange that I make the mini [repo](https://stackblitz.com/edit/vitest-dev-vitest-ym539e?file=components%2Fcollapse.vue) , but it's work fine.🤔

**Comment by cexbrayat** [comment]
@priority3 Your repro is accurate: it fails if you use Vue 3.2.45 😉

**Comment by priority3** [comment]
> @priority3 Your repro is accurate: it fails if you use Vue 3.2.45 😉

oh, I applied the corresponding version of vue(^3.2.44) and vtu(^2.2.1), but same error, let me find the way to reproduce it.🤒

**Comment by cexbrayat** [comment]
We released VTU v2.2.2 which should help with people encountering an issue with defineExpose and Vue v3.2.45
